### PR TITLE
Pull request for GCI

### PR DESCRIPTION
**Pull request by SidDevelop**
This is done for GCI

### CODE REVIEW

**Review by nikit19 — CHANGES_REQUESTED** [review]
Travis CI has failed. Please make the changes.
You need to add more fields about yourself in students.yml

**Comment by ba11b0y** [comment]
@SidDevelop Please change the title of the PR to a meaningful one.

File: _data/students.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+-name: Sidhaarth Krishnan
+github: SidDevelop
+image: sid.jpg
```
Comment (abishekvashok) [inline _data/students.yml:3]: Image name and the name you entered doesn't match

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Also the image should be 240x240px

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@SidDevelop  Please do the changes as suggested by @abishekvashok  and PR is ready to merge 👍

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Rename `1467822650826.jpg` to `sid.jpg` or vice versa.

And, maybe your image is not in correct dimensions. If so, correct that too. :+1:

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@SidDevelop  Please rename your image to `sid.jpg`, also properly indent your code and your PR is ready to merge 👍

**Review by sayak119 — CHANGES_REQUESTED** [review]
Please indent properly

**Comment by harshit98** [comment]
Hi @SidDevelop , what's the status of this pull request ?

**Review by ParthS007 — COMMENTED** [review]
@SidDevelop  Please update your PR 👍

**Review by isuruAb — COMMENTED** [review]
please resolve conflicts

**Review by ParthS007 — COMMENTED** [review]
closing this PR due to inactivity, Please create a new PR 👍

**Pull request closed without merging**